### DISCUSSION TITLE
can find thread devices by mDNS

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#8833] [Q&A] [answered]
Dear all,
I commissioned my thread device to Apple Pod, it can shows the Pod's records, but no my thread device record there, Does anyone knows that? thank you.

![5802f5196ab67f34b2b02b2c154e83f](https://user-images.githubusercontent.com/30258768/222882306-9649d88d-3cda-45e0-923f-e00882fad804.png)

**Comment by jwhui** [comment, marked as answer]
These look like Matter service instance names and host names.

For more information about publishing DNS-SD services with OpenThread, check out the [Thread Border Router codelab](https://openthread.io/codelabs/openthread-border-router).
